papaya-core/src/main/java/pl/edu/agh/papaya/model/SprintStats.java

```diff
@@ -27,11 +27,10 @@
     private Double averageCoefficientCache = 0d;
 
     public void updateCoefficient(Duration totalDeclaredTime) {
-        if (timeBurned != null && totalDeclaredTime != null) {
+        if (timeBurned != null && totalDeclaredTime != null && !totalDeclaredTime.isZero()) {
```

Comment:
Tu się wrzucało takie `10e6` do bazy jako `coefficient` i było to nieco nieczytelne jak dla mnie